ntoskrnl/cc/pin.c

```diff
@@ -286,26 +286,19 @@ CcpPinData(
         }
     }
 
-    Result = FALSE;
-    _SEH2_TRY
+    /* Ensure the pages are resident */
+    Result = CcRosEnsureVacbResident(NewBcb->Vacb,
+                                     BooleanFlagOn(Flags, PIN_WAIT),
+                                     BooleanFlagOn(Flags, PIN_NO_READ),
+                                     VacbOffset,
+                                     Length);
```

Comment:
While this function is no longer wrapped within a SEH block, it actually raises a status exception. I don't know exactly how dramatic the situation can be if a VACB couldn't be set as resident so perhaps removing `ExRaiseStatus` in that function should be more than enough?

Reply:
Yes it should just return TRUE/FALSE. Fixed that.